src/odemis/driver/technolution.py

```diff
@@ -611,8 +611,7 @@ def __init__(self, name, role, parent, **kwargs):
         # delay between the trigger signal to start the acquisition and the scanner to start scanning
         # x: delay in starting a line scan; y: delay in scanning full lines (prescan lines)
         # TODO: y scan delay is y prescan lines which is currently unused an can probably be deleted.
-        # The scanDelay in x direction maximum (200e-6) is experimentally determined.
-        self.scanDelay = model.TupleContinuous((0.0, 0.0), range=((0.0, 0.0), (200e-6, 10.0)), unit='s',
+        self.scanDelay = model.TupleContinuous((0.0, 0.0), range=((0.0, 0.0), (1.0, 10.0)), unit='s', cls=(int, float),
```

Comment:
This is really confusing: the X is in seconds, but the Y in some "number of lines". However the unit is "s". In addition, Y is actually not implemented, but it's allowed to put up to 10. 

If it's not implemented, I'd suggest to just put as maximum range "0", to make it clear it's not possible to pass anything else (and expect it to be working). If you really want to support it, then I'd suggest to accept seconds, and convert from seconds to line scan in getTicksScanDelay().